diff --git a/dist/mode-states.svelte.js b/dist/mode-states.svelte.js
index 79743820f143371f0bf946b318c33a6f28bf4082..21e9ac5325ec65dca36141b33e3fbc3f85e4b100 100644
--- a/dist/mode-states.svelte.js
+++ b/dist/mode-states.svelte.js
@@ -43,9 +43,7 @@ export class SystemPrefersMode {
     #defaultValue = undefined;
     #track = true;
     #current = $state(this.#defaultValue);
-    #mediaQueryState = typeof window !== "undefined" && typeof window.matchMedia === "function"
-        ? new MediaQuery("prefers-color-scheme: light")
-        : { current: false };
+    #mediaQueryState = typeof window !== "undefined" && typeof window.matchMedia === "function" ? new MediaQuery("prefers-color-scheme: light") : { current: false };
     query() {
         if (!isBrowser)
             return;
